pahfit/base.py

```diff
@@ -22,12 +22,14 @@
     eqws,
 )
 
+from pahfit.features import Features
```

Comment:
This fails in the tests because there is no __init__.py in pahfit/features, so pahfit.features is not seen as a module yet

Reply:
Thanks, had one locally.  I pushed an `__init__.py`.  I think it could be omitted but then we'd need `from pahfit.features.features`.  This is better.  We should for consistency across sub-packages.